Lib/test/test_venv.py

```diff
@@ -33,6 +33,12 @@
     or sys._base_executable != sys.executable,
     'cannot run venv.create from within a venv on this platform')
 
+requireAlternativePythonInPath = unittest.skipUnless(
+        shutil.which("python") is not None
```

Comment:
This won't work if `python` resolves to a Python 2.x interpreter, as they don't know about Python's built-in venv (which appeared in Python 3.3). See comment below on how test should be structured.